features/creating_order/being_unable_to_create_order_when_item_prices_are_not_defined_for_channel.feature

```diff
@@ -0,0 +1,20 @@
+@managing_orders
+Feature: Being unable to create order when item prices are not defined for channel
+    In order to have proper data provided for channel when creating an order
+    As an Administrator
+    I want to be unable to create an order when item prices are not defined for channel
+
+    Background:
+        Given the store operates on a channel named "Web-US" in "USD" currency
+        And the store operates on a channel named "Web-GB" in "USD" currency
```

Comment:
We might want to pick up countries which use USD as national currency like East Timor, Palau or Ecuador 🎉